### DISCUSSION TITLE
E3 v2

### DISCUSSION THREAD

**Discussion by jurson8204** [Jyers/Marlin#680] [Q&A] [answered]
Greetings all,
I really enjoy jyers compilation for a stock board, but I have orderer bigtreetech mini e3 v2 and would like to compile this firmware for this board. Is the only thing to change  envs for STM32F103RC_btt_512K or is it a lot more complicated than I know? I have 0 experience in programming, the only knowledge I have is from what I read throughout Internet. Thanks for replies.

**Comment by Jyers** [comment, marked as answer]
As far as I know it should be that simple, but I have not used that board before so I am not certain.

**Reply by jurson8204** [reply]
Thanks for response. I will compile it then this way. When the board arrives I will try to flash the firmware :)

**Reply by jurson8204** [reply]
WIth dealing with that for too long I have order tft35 display... I am now rebuilding marlin and the only error I have is to select only one controller option. I have defined cr10 #define CR10_STOCKDISPLAY and undefined 
    //#define DWIN_CREALITY_TOUCHLCD
    //#define CREALITY_DWIN_EXTUI
    //#define DWIN_CREALITY_LCD
What should I change to make it work?

**Reply by humanuser1** [reply]
I also have the same parameters disabled but I'm running into a similar issue below?


Marlin\src\lcd\extui\lib\creality_dwin\creality_dwin.cpp: In member function 'void CrealityDWINClass::Popup_Control()':
Marlin\src\lcd\extui\lib\creality_dwin\creality_dwin.cpp:4712:29: error: 'LEVELING_GET_MESH' was not declared in this scope
         Draw_Menu(Leveling, LEVELING_GET_MESH);

**Comment by tome9111991** [comment]
Stock display dont work plug and play with skr mini.
You have to repin

**Reply by jurson8204** [reply]
Yup, I found this out: https://github.com/bigtreetech/BIGTREETECH-SKR-mini-E3/issues/424 and I am currently trying to make changes in jyers firmware

**Reply by tome9111991** [reply]
> Yup, I found this out: [bigtreetech/BIGTREETECH-SKR-mini-E3#424](https://github.com/bigtreetech/BIGTREETECH-SKR-mini-E3/issues/424) and I am currently trying to make changes in jyers firmware

Your skr is working now with DWIN?

Maybe i want replace my board too

**Comment by jurson8204** [comment]
I am currently onboard, so I couldn't check. Anyway I abonnded the project
and ordered tft display

niedz., 9 maj 2021, 20:08 użytkownik tome9111991 ***@***.***>
napisał:
